diff --git a/build-module/advanced-filters/number-filter.js b/build-module/advanced-filters/number-filter.js
index 78994fb763dcd7cf665e481f825fb322e233bd58..21709001564a8bb77ec52abe92097a8205d09ed1 100644
--- a/build-module/advanced-filters/number-filter.js
+++ b/build-module/advanced-filters/number-filter.js
@@ -6,7 +6,7 @@ import { SelectControl, TextControl } from '@wordpress/components';
 import { get, find, isArray } from 'lodash';
 import clsx from 'clsx';
 import { sprintf, __, _x } from '@wordpress/i18n';
-import { CurrencyFactory } from '@woocommerce/currency';
+import CurrencyFactory from '@woocommerce/currency';
 /**
  * Internal dependencies
  */
diff --git a/build-module/experimental.js b/build-module/experimental.js
index 7b6ec5b389beac67ca597563037a7a8eb0850bf4..a49e4c30cf685841777e54c71b9940c74876cd12 100644
--- a/build-module/experimental.js
+++ b/build-module/experimental.js
@@ -1,9 +1,9 @@
 /**
  * External dependencies
  */
-import { __experimentalText, Text as TextComponent, } from '@wordpress/components';
+import { __experimentalText, } from '@wordpress/components';
 /**
  * Export experimental components within the components package to prevent a circular
  * dependency with woocommerce/experimental. Only for internal use.
  */
-export const Text = TextComponent || __experimentalText;
+export const Text = __experimentalText;
diff --git a/build-module/filters/index.js b/build-module/filters/index.js
index d69a26f916b1ef064ed1141b699406c69eeec5e1..6a6f8f027b952d748ba0616417ed1be0b43d9ff0 100644
--- a/build-module/filters/index.js
+++ b/build-module/filters/index.js
@@ -7,7 +7,7 @@ import { find } from 'lodash';
 import PropTypes from 'prop-types';
 import { updateQueryString } from '@woocommerce/navigation';
 import { getDateParamsFromQuery, getCurrentDates } from '@woocommerce/date';
-import { CurrencyFactory } from '@woocommerce/currency';
+import CurrencyFactory from '@woocommerce/currency';
 /**
  * Internal dependencies
  */
diff --git a/build-style/style.css b/build-style/style.css
index 6395671d5750c0cb1bb87a0da866c5e0b9a81ae9..e1ee71f2cceaf55e5d7d3c0656a47ebc8b226232 100644
--- a/build-style/style.css
+++ b/build-style/style.css
@@ -157,7 +157,7 @@
 }
 .SingleDatePicker_closeButton:focus,
 .SingleDatePicker_closeButton:hover {
-  color: darken(#cacccd,10%);
+  color: #b0b3b4;
   text-decoration: none
 }
 .SingleDatePicker_closeButton_svg {
@@ -884,7 +884,7 @@
 }
 .DateRangePicker_closeButton:focus,
 .DateRangePicker_closeButton:hover {
-  color: darken(#cacccd,10%);
+  color: #b0b3b4;
   text-decoration: none
 }
 .DateRangePicker_closeButton_svg {
